src/index.jsx

```diff
@@ -97,6 +106,36 @@ async function init() {
     });
   }
 
+  // setup Grafana Faro
+  const history = createBrowserHistory();
+  let conditionalSampleRate = ddKnownBotRegex.test(navigator.userAgent) ? 0 : faroSampleRate;
```

Comment:
what does `ddKnownBotRegex.test(navigator.userAgent)` do? could you add a comment to explain 🙏 

and is it really a datadog config or is it generic and will be renamed when we remove dd?

Reply:
it will be removed when we remove DD, I will update this